semp/getVpnSpoolSemp1.go

```diff
@@ -29,7 +35,7 @@ func (e *Semp) GetVpnSpoolSemp1(ch chan<- prometheus.Metric, vpnFilter string) (
 		} `xml:"execute-result"`
 	}
 
-	command := "<rpc><show><message-spool><vpn-name>" + vpnFilter + "</vpn-name></message-spool></show></rpc>"
+	command := "<rpc><show><message-spool><vpn-name>" + vpnFilter + "</vpn-name><detail/></message-spool></show></rpc>"
```

Comment:
Please provide some time measurements. 
How much taker 100 calls with <details> and 100 without.
To estimate if this changes something about the broker performance risk.

Reply:
ok. I did that in a shell script. With millisecond granularity the timing to make 100 calls were within 8 milliseconds of each other. Suspect that the event broker has the info at its fingertips and just populates the xml with the extra info. The event broker was running in a docker container.